meta-client/src/main/java/org/apache/helix/metaclient/recipes/leaderelection/LeaderElectionClient.java

```diff
@@ -69,6 +69,8 @@ public class LeaderElectionClient implements AutoCloseable {
   private Set<String> _leaderGroups = new HashSet<>();
 
   private final static String LEADER_ENTRY_KEY = "/LEADER";
+  private final static String PARTICIPANTS_ENTRY_KEY = "/PARTICIPANTS";
+  private final static String PARTICIPANTS_ENTRY_PARENT = "/PARTICIPANTS/";
```

Comment:
NIT: this is a little duplicated.  We can pick the KEYWORD = "PARTICIPANTS" and use KEYWORD create the parent path.